packages/xstate-solid/package.json

```diff
@@ -53,6 +53,6 @@
   "devDependencies": {
     "solid-js": "^1.7.6",
     "solid-testing-library": "^0.3.0",
-    "xstate": "5.17.1"
+    "xstate": "workspace:^"
```

Comment:
I've updated all `xstate` versions to use the `workspace:` protocol.

This should reduce the number of merge conflicts we need to resolve due to bumped versions.